router/httpTriggers.go

```diff
@@ -49,17 +49,20 @@ type HTTPTriggerSet struct {
 	functions         []crd.Function
 	funcStore         k8sCache.Store
 	funcController    k8sCache.Controller
+
+	tsRoundTripperParams tsRoundTripperParams
 }
 
 func makeHTTPTriggerSet(fmap *functionServiceMap, fissionClient *crd.FissionClient,
-	kubeClient *kubernetes.Clientset, executor *executorClient.Client, crdClient *rest.RESTClient) (*HTTPTriggerSet, k8sCache.Store, k8sCache.Store) {
+	kubeClient *kubernetes.Clientset, executor *executorClient.Client, crdClient *rest.RESTClient, params tsRoundTripperParams) (*HTTPTriggerSet, k8sCache.Store, k8sCache.Store) {
```

Comment:
Very minor comment, but it's better to pass a `*tsRoundTripperParams` (pointer instead of copy of struct)